edx_django_utils/monitoring/tests/test_custom_monitoring.py

```diff
@@ -151,37 +151,3 @@ def test_deprecated_set_custom_attributes_for_course_key(self, mock_set_custom_a
     def test_record_exception(self, mock_record_exception):
         record_exception()
         mock_record_exception.assert_called_once()
-
-
-class TestDeploymentMonitoringMiddleware(TestCase):
```

Comment:
**Note to reviewer**: [Mark resolved once seen.] This test class was moved to the other file for more consistency.